build.xml

```
@@ -0,0 +1,188 @@
+                failonerror="false">
+            <fileset dir="${lib.dir}"
+                     defaultexcludes="false"/>
+            <fileset dir="${classes.dir}"
+                     defaultexcludes="false"/>
+            <fileset dir="${build.dir}"
+                     defaultexcludes="false"/>
+        </delete>
+    </target>
+
+    <target name="compile-test"
+            depends="init,retrieve">
+        <mkdir dir="${classes.dir}/test"/>
+        <depend srcDir="${test.src.dir}"
+                destDir="${classes.dir}/test"
+                cache="${classes.dir}"/>
+        <dependset>
+            <srcfilelist files="build.xml"/>
+            <srcfilelist files="ivy.xml"/>
+            <srcfilelist files="ivysettings.xml"/>
+            <targetfileset dir="${classes.dir}/test"/>
+        </dependset>
+        <taskdef resource="org/codehaus/groovy/antlib.xml"
+                 uri="antlib:org.codehaus.groovy.ant">
+            <classpath>
+                <fileset dir="${lib.dir}/groovy-ant"
+                         includes="*.jar"/>
+            </classpath>
+        </taskdef>
+        <groovy:groovyc srcdir="${test.src.dir}"
+                        destdir="${classes.dir}/test"
+                        encoding="UTF-8"
+                        includeAntRuntime="false"
+                        fork="true">
+            <groovy:classpath id="classpath.test">
+                <fileset dir="${lib.dir}/test"
+                         includes="*.jar"/>
+            </groovy:classpath>
+        </groovy:groovyc>
+    </target>
+
+    <target name="test"
+            depends="init,retrieve,compile-test"
+            description="run unit tests">
+        <delete dir="${build.dir}/test/raw-reports"/>
+        <mkdir dir="${build.dir}/test/raw-reports"/>
+        <junitlauncher printsummary="true"
+                       failureproperty="tests.failed">
+            <classpath refid="classpath.test"/>
+            <classpath location="§{test.resource.dir}"/>
+            <classpath location="${classes.dir}/test"/>
+            <classpath location="${ant.library.dir}/ant.jar"/>
+            <classpath location="${ant.library.dir}/ant-junitlauncher.jar"/>
+            <listener type="legacy-xml"
+                      outputDir="${build.dir}/test/raw-reports"/>
+            <testclasses>
+                <fork includeJUnitPlatformLibraries="false"
+                      includeAntRuntimeLibraries="false"/>
+                <fileset dir="${classes.dir}/test">
+                    <scriptselector language="groovy">
+                        <classpath>
+                            <fileset dir="${lib.dir}/groovy-script"
+                                     includes="*.jar"/>
+                        </classpath>
+                        <![CDATA[
+                            import org.spockframework.buildsupport.SpecClassFileFinder
+                            self.selected = new SpecClassFileFinder().isRunnableSpec(file)
```

Why not a single line
```suggestion
                            self.selected = new org.spockframework.buildsupport.SpecClassFileFinder().isRunnableSpec(file)
```

---

No specific reason, just two shorter lines instead of one long line.
If you want one long line instead, I'll change it.